src/classes/CW_MarcomBriefViewController.cls

```diff
@@ -0,0 +1,313 @@
+							[select Asset_Type__r.Id, Asset_Type__r.Name from CW_Campaign_Asset_Type__c where Campaign__c = :camp.Id],
+							ConstantUtil.CW_CAT_REL_ASSETTYPE,ConstantUtil.SOBJECT_NAME,ConstantUtil.SOBJECT_ID,ConstantUtil.SOBJECT_ID)){
+				tempList.Add(rec);
+				if(tempList.size()>6){
+					assetTypeOptions.Add(tempList);
+					tempList=new List<WrapperUtil.selectWrapper>();
+				}
+			}
+			if(tempList.size()>0)
+					assetTypeOptions.Add(tempList);
+		assetTypeId = assetTypeOptions.isEmpty() ? null : (Id) assetTypeOptions.get(0).get(0).optionValue;
+	}
+    
+    //query all the campaign fields
+  	private CW_Campaign__c getCampaignInfo() {
+        String queryString = 'SELECT ';
+        for (Schema.SObjectField f : Schema.SObjectType.CW_Campaign__c.fields.getMap().values())
+        {
+            queryString += f.getDescribe().getName() + ', ';
+        }
+        //queryString = queryString.removeEnd(', ');
+        queryString += 'Owner.Name FROM CW_Campaign__c WHERE Id =\'' + camp.Id + '\'';
+        
+        return Database.query(queryString);
+    }
+    
+    public pageReference editRecord()
+    {
+    	PageReference ref = Page.CW_WizardStartPageSelector;
+    	ref.getParameters().put(ConstantUtil.SOBJECT_ID,camp.Id);
+    	ref.getParameters().put(ConstantUtil.PAGEPARAM_ISBRIEFVIEWMODE,'False');
+    	return ref;
+    }
+    
+    public pageReference deleteRecord()
+    {
+    	try 
+    	{
+    		delete camp;
+    	
+	    	PageReference pageRef = new PageReference('/' + Schema.sObjectType.CW_Campaign__c.keyPrefix + '/o');    
+	    	pageRef.setRedirect(true);
+	    	return pageRef; 
+	    	
+    	}catch(Exception ex){
+			ApexPages.addMessages(ex);
+		}
+		
+		return null;
+    }
+    
+    public pageReference backToListView()
+    {
+    	PageReference pageRef = new PageReference('/' + Schema.sObjectType.CW_Campaign__c.keyPrefix + '/o');    
+    	pageRef.setRedirect(true);
+    	return pageRef; 
+    }
+    
+    public List<WagerUplift> getWagerUplift() {
+            return getWagerUpliftFields();
+    }
+    
+    public List<NGRUplift> getNGRUplift() {
+            return getNGRUpliftFields();
+    }
+    
+    public List<WagerUplift> getWagerUpliftFields() 
+   	{
+        List<WagerUplift> WagerUpliftFields = new List<WagerUplift>();
+
+        CW_Campaign__c dataItems = [SELECT Average_Wager_Uplift_Control__c, 
+        								   Average_Wager_Uplift_Target__c,
+        								   Average_NGR_Uplift_Control__c,
+        								   Average_NGR_Uplift_Target__c 
+        							FROM CW_Campaign__c
+        							WHERE Id=: camp.Id];
+        
+       WagerUplift uplift = new WagerUplift ('Control', dataItems.Average_Wager_Uplift_Control__c);
+       WagerUpliftFields.add(uplift);
+       
+       uplift = new WagerUplift ('Target', dataItems.Average_Wager_Uplift_Target__c);
+       WagerUpliftFields.add(uplift);
+       
+       return WagerUpliftFields;
+    }
+    
+    public List<NGRUplift> getNGRUpliftFields() 
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.